cms/djangoapps/contentstore/views/tests/test_videos.py

```diff
@@ -519,6 +536,262 @@ def test_video_status_update_request(self, mock_logger):
+            upload_response: Upload response object
+
+        Returns:
+            uploaded image url
+        """
+        self.assertEqual(upload_response.status_code, 200)
+        response = json.loads(upload_response.content)
+        val_image_url = get_course_video_image_url(course_id=course_id, edx_video_id=edx_video_id)
+        self.assertEqual(response['image_url'], val_image_url)
+
+        return val_image_url
+
+    def verify_error_message(self, response, error_message):
+        """
+        Verify that image upload failure gets proper error message.
+
+        Arguments:
+            response: Response object.
+            error_message: Expected error message.
+        """
+        self.assertEqual(response.status_code, 400)
+        response = json.loads(response.content)
+        self.assertIn('error', response)
+        self.assertEqual(response['error'], error_message)
+
+    def test_video_image(self):
+        """
+        Test video image is saved.
+        """
+        edx_video_id = 'test1'
+        video_image_upload_url = self.get_url_for_course_key(self.course.id, {'edx_video_id': edx_video_id})
+        with make_image_file(
+            dimensions=(settings.VIDEO_IMAGE_MIN_WIDTH, settings.VIDEO_IMAGE_MIN_HEIGHT),
+        ) as image_file:
+            response = self.client.post(video_image_upload_url, {'file': image_file}, format='multipart')
+            image_url1 = self.verify_image_upload_reponse(self.course.id, edx_video_id, response)
+
+        # upload again to verify that new image is uploaded successfully
+        with make_image_file(
+            dimensions=(settings.VIDEO_IMAGE_MIN_WIDTH, settings.VIDEO_IMAGE_MIN_HEIGHT),
+        ) as image_file:
+            response = self.client.post(video_image_upload_url, {'file': image_file}, format='multipart')
+            image_url2 = self.verify_image_upload_reponse(self.course.id, edx_video_id, response)
+
+        self.assertNotEqual(image_url1, image_url2)
+
+    def test_video_image_no_file(self):
+        """
+        Test that an error error message is returned if upload request is incorrect.
+        """
+        video_image_upload_url = self.get_url_for_course_key(self.course.id, {'edx_video_id': 'test1'})
+        response = self.client.post(video_image_upload_url, {})
+        self.verify_error_message(response, 'No file provided for video image')
+
+    def test_invalid_image_file_info(self):
+        """
+        Test that when no file information is provided to validate_video_image, it gives proper error message.
+        """
+        error = validate_video_image({})
+        self.assertEquals(error, 'The image must have name, content type, and size information.')
+
+    def test_currupt_image_file(self):
+        """
+        Test that when corrupt file is provided to validate_video_image, it gives proper error message.
+        """
+        with open(MEDIA_ROOT + '/test-corrupt-image.png', 'w+') as file:
```

Comment:
@mushtaqak Where is this file? I don't see it anywhere?

Also can we avoid use of `MEDIA_ROOT` here directly? This is only file except the setting files where we are using `MEDIA_ROOT`.